bin/setup_dev

```diff
@@ -168,6 +169,11 @@ else
     (cd forch; git fetch; git checkout $FORCHV)
 fi
 
+SITE_PACKAGES_PATH=$(python3 -c 'import sysconfig; print(sysconfig.get_paths()["purelib"])')
```

Comment:
where is this?  Is this in /usr/lib or something like that?  If so, we really shoudn't be futzing with it...

Reply:
Reverted this file. Seems that the change is not useful..